tests/test_bugs.py

```diff
@@ -0,0 +1,17 @@
+from selenium.webdriver.common.by import By
```

Comment:
Is this test necessary? We want to test our features and not the bug that is present

Reply:
I don't think having tests for the bugs will be all bad, this way we can make sure the bug was fixed properly

Reply:
Okay, thats fine, we can leave it then . Ill approve the PR since the tests were written properly